### PR TITLE
Add more debug log in join.

### PR DESCRIPTION
**Pull request by gyliu513**
/cc @marun @shashidharatd @onyiny-ang

### CODE REVIEW

File: pkg/kubefnord/join.go [hunk 1]
```diff
@@ -250,7 +250,7 @@ func performPreflightChecks(clusterClientset client.Interface, name, host,
 	} else if err != nil {
 		return err
 	} else if sa != nil {
-		return fmt.Errorf("service account already exists in joining cluster")
+		return fmt.Errorf("service account %s already exists in joining cluster: %s", saName, host)
```
[the remark this answers is not in the document]
Comment (irfanurrehman) [reply pkg/kubefnord/join.go]: A colon preferred after service account also.
`service account: %s`

File: pkg/kubefnord/join.go [hunk 2]
```diff
@@ -267,15 +267,16 @@ func addToClusterRegistry(hostConfig *rest.Config, host, joiningClusterName stri
 		return err
 	}
 
-	glog.V(2).Info("Registering cluster with the cluster registry.")
+	glog.V(2).Info("Registering cluster %s with the cluster registry.", joiningClusterName)
 
 	_, err = registerCluster(crClientset, host, joiningClusterName, dryRun)
 	if err != nil {
-		glog.V(2).Infof("Could not register cluster with the cluster registry: %v", err)
+		glog.V(2).Infof("Could not register cluster %s with the cluster registry: %v",
```
Comment (irfanurrehman) [inline pkg/kubefnord/join.go]: `Could not register cluster: %s`

File: pkg/kubefnord/join.go [hunk 3]
```diff
@@ -401,30 +402,32 @@ func createRBACSecret(hostClusterClientset, joiningClusterClientset client.Inter
...
 
-	glog.V(2).Info("Creating service account in joining cluster")
+	glog.V(2).Info("Creating service account in joining cluster: %s", joiningClusterName)
 
 	saName, err := createServiceAccount(joiningClusterClientset, namespace,
 		joiningClusterName, hostClusterName, dryRun)
 	if err != nil {
-		glog.V(2).Infof("Error creating service account in joining cluster: %v", err)
+		glog.V(2).Infof("Error creating service account %s in joining cluster: %v",
+			saName, err)
```
Comment (irfanurrehman) [inline pkg/kubefnord/join.go]: Why not use the `joiningClusterName` also here to be consistent with other prints.

File: pkg/kubefnord/join.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (gyliu513) [reply pkg/kubefnord/join.go]: Done

File: pkg/kubefnord/join.go [hunk 4]
```diff
@@ -267,15 +267,16 @@ func addToClusterRegistry(hostConfig *rest.Config, host, joiningClusterName stri
 		return err
 	}
 
-	glog.V(2).Info("Registering cluster with the cluster registry.")
+	glog.V(2).Info("Registering cluster %s with the cluster registry.", joiningClusterName)
```
[the remark this answers is not in the document]
Comment (gyliu513) [reply pkg/kubefnord/join.go]: Done

File: pkg/kubefnord/join.go (hunk 2 above)
Comment (gyliu513) [reply pkg/kubefnord/join.go]: Done

File: pkg/kubefnord/join.go (hunk 3 above)
Comment (gyliu513) [reply pkg/kubefnord/join.go]: Done

**Pull request merged**